File: Rust/7 Kyu/lost_number_in_number_sequence.rs
fn find_deleted_number(list: &[u16], mixed_list: &[u16]) -> Option<u16> {
    let a:u16 = list.iter().sum();
    let b:u16 = mixed_list.iter().sum();
    let soma:u16 = a-b;
    if soma ==0{
        return  None;
    }
    Some(soma)

}

File: CPP/7 Kyu/lost_number_in_number_sequence.cpp
using namespace std;
#include <numeric>


int findDeletedNumber(list<int> startingList, list<int> mixedList)
{
   int a =  accumulate(startingList.begin(), startingList.end(), 0);
   int b =  accumulate(mixedList.begin(), mixedList.end(), 0);
  
  return a-b;

}